### DISCUSSION TITLE
Tags input for next.js

### DISCUSSION THREAD

**Discussion by muhammadimran787898** [shadcn-ui/ui#3053] [Ideas]
tags input should be add in next js  component

**Comment by Abdul-Manan-456** [comment]
i read it throughly.this component should be added.

**Comment by ansar-akbar** [comment]
Hello Shade CN 

I hope this message finds you well. I am reaching out to suggest an enhancement for our toolkit: the addition of a Tag Input Component with nextjs. This component would greatly benefit users who need an efficient way to input and display tags for categorization or filtering purposes. Its inclusion could enhance user experience by providing a more interactive and user-friendly interface for managing tags.

Thank you for considering my suggestion. I look forward to seeing our toolkit grow and improve with contributions from our vibrant community.

**Comment by syedslegend786** [comment]
Yeh this should be added.

**Comment by kavinda1995** [comment]
For those are keen on over the self solution - try https://emblor.jaleelbennett.com/

**Reply by JoeBuildsStuff** [reply]
this
